adapters/eea-direct.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+import { acceptableParameters, convertUnits } from '../lib/utils';
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+import { parallel, map, filter } from 'async';
+import { parse } from 'babyparse';
```

Comment:
Actually, we are already using `csv-parse` elsewhere. Is there a good reason to use `babyparse` instead?